### PR TITLE
Mark/create form component

### PR DESCRIPTION
**Pull request by markwuu**

### CODE REVIEW

**Comment by markwuu** [comment]
[Implement styled form input](https://trello.com/c/jgdXO3Gd/31-implement-styled-form-input)

File: src/components/Form.tsx [hunk 1]
```diff
@@ -1,6 +1,39 @@
...
+  color: #fff;
+  font-family: Roboto;
+  font-size: 12px;
+  letter-spacing: 1px;
+  padding-bottom: 15px;
+  padding-top: 15px;
+  text-align: center;
+`
+
+const Input = styled.input`
```
Comment (GRardB) [inline src/components/Form.tsx]: Move this to its own file: `src/components/UI/Form/Input.tsx`,

File: src/components/Container/index.tsx [hunk 2]
```diff
@@ -1,6 +1,9 @@
 import styled from 'styled-components'
 
 export const Container = styled.div`
+  border: 2px solid #ccc;
+  border-radius: 4px;
+  display: inline-block;
```
[the remark this answers is not in the document]
Comment (markwuu) [reply src/components/Container/index.tsx]: screen shot 2018-08-29 at 9 45 21 pm
This is the form component without those styles above. But like you said this is to make the form input component and not the whole form component itself.

[the remark this answers is not in the document]
Comment (GRardB) [reply src/components/Container/index.tsx]: The purpose of `Container` is not for the form, but for making sure that specific content is centered and doesn't exceed 1400px. It's a fairly common thing in CSS frameworks ([e.g. bootstrap](https://getbootstrap.com/docs/4.1/layout/overview/#containers)).

I would also say that as a general rule, you don't want to edit styles for components unless you know that you want to change them everywhere. You have to be careful when modifying existing styles and make sure that everything else looks as intended. With these changes, this is what the landing page looks like:

react_app

A better idea in this scenario would be to have a new styled component which took care of the sizing of the form, or styling the form itself.

File: src/components/LandingPage/Masthead.tsx [hunk 3]
```diff
@@ -23,7 +23,7 @@ const Subheading = styled.p`
 
 export const Masthead = () => (
   <UIMasthead>
-    <Heading>Too many TV shows to track?</Heading>
+    <Heading>Too many TV shows to track?</Heading>x
```
Comment (GRardB) [inline src/components/LandingPage/Masthead.tsx]: why lol

**Pull request merged**